kalite/django_cherrypy_wsgiserver/cherrypyserver.py

```diff
@@ -142,7 +142,12 @@ def run_cherrypy_server(host="127.0.0.1", port=None, threads=None, daemonize=Fal
         # stop_server(pidfile)
 
         from django.utils.daemonize import become_daemon
-        become_daemon()
+        kalite_home = os.environ.get("KALITE_HOME", None)
+        logfile = os.path.join(kalite_home, "kalite.log") if (kalite_home and os.environ.get("NAIVE_LOGGING", False)) else None
```

Comment:
Should we default this to `True` instead? If we can have the logger log on both a file and stdout, that would be great.